suites/nautilus/rgw/tier-1_containerized_rgw_ssl_regression-suite.yaml

```diff
@@ -0,0 +1,107 @@
+tests:
```

Comment:
Could we add some doc strings like the purpose, conf file to be used, etc? Also, can we rename the file to `tier-1_rgw_ssl_containerized-regression.yaml`

Reply:
Made changes